README.md

```diff
@@ -202,3 +202,46 @@ Copyright 2022-2023 Mina Foundation, Inc.
 
 Free use of this software is granted under the terms of the Mozilla
 Public License 2.0.
+
+### Quick Start Guide
+
+Follow these steps to quickly set up and run the Mina Indexer on your machine.
+
+1. **Clone the Repository**
+
+Open your terminal and run the following commands:
+```sh
+git clone git@github.com:Granola-Team/mina-indexer.git
+cd mina-indexer
+```
+
+2. **Build the Indexer**
+
+Build the project using one of the following methods:
+* Using Cargo:
+```sh
+cargo build --release
+```
+
+Using Nix (Recommended):
+```sh
+nix develop
+nix build '.?submodules=1'
+```
+
+Using Docker:
+```sh
+docker build -t mina-indexer:latest .
+```
+
+3. **Run the Indexer**
+
+Using Nix as an example, in your initial terminal window where you ran the aforementioned Nix commands, now run:
+```sh
+RUST_BACKTRACE=1 target/release/mina-indexer server -l tests/data/genesis_ledgers/mainnet.json -s /path/to/setup_directory_containing_blocks
```

Comment:
don't need `RUST_BACKTRACE`, also add `result/bin/mina-indexer` for `nix build`

Reply:
addressed in new commit below